package.json

```diff
@@ -28,6 +28,9 @@
   "bugs": {
     "url": "https://github.com/merencia/node-cron/issues"
   },
+  "dependencies": {
+    "uuid": "^8.3.2"
```

Comment:
can you pin the dependency to exact version.

Reply:
Fixed. Please take a look again. Thank you.